### ISSUE TITLE
Cannot query the Google Doc's set description values with graphql

### ISSUE THREAD

**Issue by linoxyn** [cedricdelpoux/gatsby-source-google-docs#71]
I have installed the latest `gatsby-source-google-docs@2.0.0-beta.19`  
On `gatsby develop` all my queries now fail pertaining to all the Doc's **description** values.

I believe now we are to make sure our description written in YAML format and not JSON, is this the case?

If so do we need to use `---` at the head of the YAML format?

I have found that in `graphql` I cannot query for any specific values within the YAML structure. I can only return a sting of the description: 

Query:
```
{
  allMarkdownRemark {
    edges {
      node {
        frontmatter {
          description
        }
      }
    }
  }
}
```
There are no sub-fields within the `description` field.

Example of one of the returned nodes showing the description field value:

```
{
          "node": {
            "frontmatter": {
              "description": "page: 'exhibition'\n                           title: February Feature\n                           showstart: 2020-02-08\n                           showend: 2020-02-15\n                           showlocation: Toronto\n                           showcode: 2020020820200215TOR\n                           showstatus: upcoming\n                           thumbalt: Multimedia sculpture Rabbit by Patrick Amiot"
            }
          }
        },
```

Thank you

**Comment by cedricdelpoux** [comment]
Yes sorry for the change. 
I removed the JSON support because YAML is so much easier to write. 

You juste need to write:
```yaml
prop: value
```

Values can be integer, boolean, date, string... You even don't need to use quotes for string values. So much easier in my opinion. What do you think?

**Comment by linoxyn** [comment]
@cedricdelpoux 
Yes, YAML is easier, and I have changed all my Google Sheets App Scripts to now auto generate the YAML structure in the description field in the Google Docs. Though now in running gatsby I can now no longer access the separate values within the `description` For example I would like to access the values in graphql like this:
```
{
  allMarkdownRemark {
    edges {
      node {
        frontmatter {
          description {
             page
             title
             showstart
             showlocation
             showcode
             showstatus
             thumbalt
          }
        }
      }
    }
  }
}
```
The values in the Google Docs description are not accessible by graphql under `frontmatter` as they were before, or under the new field `description`. It is only returning a string.

**Comment by cedricdelpoux** [comment]
Could you try to remove `description` from the query?

```
{
  allMarkdownRemark {
    edges {
      node {
        frontmatter {
             page
             title
             showstart
             showlocation
             showcode
             showstatus
             thumbalt
        }
      }
    }
  }
}
```

**Comment by linoxyn** [comment]
Here is a screenshot in `graphiql`

![plugin-docs](https://user-images.githubusercontent.com/29573699/93220574-b5e3f200-f73a-11ea-9223-9f0dc8b40add.jpg)

I've highlighted `frontmatter/page` and `frontmatter/title` Are these now internal fields? I did not have conflicts with these before. I am not sure if that is causing the issue?

Could my YAML structures be incorrectly written? Here is one example from one of my Google Docs:

```
page: exhibition
title: Virtual Spring Exhibition
showstart: 2020-03-24
showend: 2020-05-30
showlocation: Toronto Calgary Winnipeg
showcode: 2020032420200530TOR
showstatus: current
thumbalt: John Hall - Framed: Magdalena Rosa
```

The errors on `gatsby develop`  all list the failures of finding field values within the frontmatter descriptions: One example:

```
ERROR #85923  GRAPHQL

There was an error in your GraphQL query:

Cannot query field "imgalt" on type "MarkdownRemarkFrontmatter".

If you don't expect "imgalt" to exist on the type "MarkdownRemarkFrontmatter" it is most likely a typo.
However, if you expect "imgalt" to exist there are a couple of solutions to common problems:
```

**Comment by cedricdelpoux** [comment]
You expect `imgalt` but you specified `thumbalt` in your YAML. Is it a typo?

I can't reproduce your issue. I tried to paste your YAML description into one of my documents.
<img width="1270" alt="Screenshot 2020-09-16 at 08 43 53" src="https://user-images.githubusercontent.com/2683300/93301536-e38f7080-f7f8-11ea-9212-8ca91899f021.png">

**Comment by linoxyn** [comment]
@cedricdelpoux Thank you for your attention in trying to help me out. In the end I have found the issues, the main one being that the changes I made to my Google Sheets App Scripts that functionally created the YAML in the decsription area of each Doc created, and even though the formatting presented correctly, when I clicked the edit icon beside the description I then found that the lining up of the YAML was actually not correct. I also had to fix some of the entries into the YAML that come from the sheets, in the queries, for example date formatting and boolean values which Google Sheets uses it's own date and boolean formatting. 

All is working well now. Again, thank you  😊
